### DISCUSSION TITLE
Portal engine stats error

### DISCUSSION THREAD

**Discussion by mcg-pimcore** [pimcore/pimcore#10854] [Q&A / Support]
Hello,

Issue is coming on following portal-engine request. Here to mention that pimcore version is 10.0.6 and using basic (es_client) authentication for elastic search. Also facing same issue on assets download.

http://domain.com/_portal-engine/stats/portal/data?dataSource=pimcore_portal_engine_statistics_explorer_download&statisticMode=statistic&rows=system_fields.id%2Cpath&columns=&fieldSettings=%7B%22system_fields.id%22%3A%7B%22typeGroup%22%3A%22default%22%2C%22order%22%3A%22_count%7C%7Cdesc%22%2C%22max%22%3A10%7D%7D&configurationId=global_most_downloaded_assets&filters=%5B%5D

{"success":false,"message":"Error loading data: {\u0022error\u0022:{\u0022root_cause\u0022:[{\u0022type\u0022:\u0022security_exception\u0022,\u0022reason\u0022:\u0022missing authentication credentials for REST request [\/devstatistics_download\/_search]\u0022,\u0022header\u0022:{\u0022WWW-Authenticate\u0022:\u0022Basic realm=\\\u0022security\\\u0022 charset=\\\u0022UTF-8\\\u0022\u0022}}],\u0022type\u0022:\u0022security_exception\u0022,\u0022reason\u0022:\u0022missing authentication credentials for REST request [\/devstatistics_download\/_search]\u0022,\u0022header\u0022:{\u0022WWW-Authenticate\u0022:\u0022Basic realm=\\\u0022security\\\u0022 charset=\\\u0022UTF-8\\\u0022\u0022}},\u0022status\u0022:401}"}

**Comment by fashxp** [comment]
the basic authentication is not configurable yet for statistics explorer, see also https://github.com/pimcore/statistics-explorer/issues/31 (only available for enterprise customers). 

either wait for implementation of this issue or deactivate the statistics tracking for now (or don't use authentication for elastic search for now, but make sure cluster is only accessible by permitted resources).
